src/Item/Item.service.ts

```diff
@@ -138,7 +138,10 @@ export class ItemService {
 
     const dbItem = await Item.findOne<ItemAttributes>(item.id)
     if (dbItem) {
+      item.updated_at = new Date()
       this.checkItemIsMovedToAnotherCollection(item, dbItem)
+    } else {
+      item.created_at = new Date()
```

Comment:
I think updated_at will be set here regardless, by the decentraland-server lib (to avoid having a NULL updated_at timestamp) maybe we can set it here too?